app/helpers/general-date.js

```diff
@@ -8,7 +8,7 @@ export function generalDate(params) {
 
   const local = moment(params[0]).tz(timezone).locale();
 
-  let format = params[2] || 'h:mm A, MMMM Do YYYY (z)';
+  let format = params[2] || 'dddd, D MMMM, YYYY h:mm A (z)';
```

Comment:
We can't change the default param just like that, it'll mess up the system elsewhere

Reply:
Ok I will revert it.

Reply:
No